dataland-backend/src/main/resources/application.properties

```diff
@@ -5,4 +5,5 @@ management.endpoint.health.show-details=always
 skyminder.user-name=${SKYMINDER_USER}
 skyminder.base-path=${SKYMINDER_URL}
 skyminder.password=${SKYMINDER_PW}
+dataland-edc-client.baseUrl=http://edc-dummyserver:8080
```

Comment:
das darf hier auf keinen fall auf dummyserver zeigen - das ist ja die properties-fiel, was auch in den deployten versionen benutzt wird. Und in denen ist das hoffentlich kein dummyserver :-)

Vorschläge: 
a) wir zeigen z.B. auf http://dataland-edc:8080, und in irgendeinem anderen Profil überschreiben wir das.
b) wir zeigen z.B. auf http://dataland-edc:8080, und je nach dem starten wir unter dem service namen eben den richtigen, oder den dummyserver
c) wir machen das wie darüber per Env-Variablen

ich glaube, jetzt wo ich es ausschreibe, mir gefällt b) am besten.

Reply:
Ich kümmer mich darum

Reply:
Option b) ist implementiert

Reply:
war dann wohl kein "qutsch" :D